packages/postgres-database/src/simcore_postgres_database/models/services.py

```diff
@@ -69,6 +70,13 @@
         onupdate=func.now(),
         doc="Timestamp with last update",
     ),
+    sa.Column(
+        "deprecated",
+        sa.DateTime(),
```

Comment:
here you use datetime... you probably need to re-generate the alembic migration code.